src/app/workshops/workshops.component.html

```diff
@@ -5,7 +5,7 @@
             (locationFilterChanged)="setLocationIdList($event)" (categoryFilterChanged)="setCategoryList($event)" (applyFilters)="toggleFilterDropdown($event)"></workshop-filter>
     </div>
     <div class="list">
-        <workshops-list [activePage]="pageNumber"></workshops-list>
+        <workshops-list [activePage]="pageNumber" [itemLimit]="0"></workshops-list>
```

Comment:
0?

Reply:
yep.. this is a required property and we want to show the page numbers for all

Reply:
moved it to the script